lib/playback/index.ts

```diff
@@ -100,8 +100,8 @@ export class Player {
 	}
 
 	async #runTrack(track: Mp4Track) {
-		if (track.kind !== "audio" && track.kind !== "video") {
-			throw new Error(`unknown track kind: ${track.kind}`)
+		if (!(track.name.toLowerCase().includes("audio")) && !(track.name.toLowerCase().includes("video"))) {
```

Comment:
`isAudioTrack(track)` or `isVideoTrack(track)` is the way you're supposed to determine the type now.

Reply:
Addressed